tests/sqllogictests/suites/stage/formats/parquet/parquet_field_types.test

```diff
@@ -182,7 +182,7 @@ parquet/json_string.parquet 4 0 NULL NULL
 query 
 select * from t2 order by a
 ----
-1 {"k1":"v1","k2":"v2"}
-2 {"k":{"k1":123,"k2":456}}
-3 [1,2,3]
-4 ["a","b"]
+1 "{\"k1\":\"v1\",\"k2\":\"v2\"}"
```

Comment:
binary output should not be quoted.

Reply:
it's string output, which is a complete string of json.

Reply:
But in logicaltests, we don't quote string output. Is this change causing incompatibility?

Reply:
oh, it's variant output with jsonb::Value::String. So it make sense. 
Anyway, a binary should neither be parsed to a json value nor shown as utf8 string, since that I've updated the test case.